interna/front/templatetags/commonmark.py

```diff
@@ -0,0 +1,18 @@
+import commonmark
+
+from django import template
+from django.utils.safestring import mark_safe
+
+register = template.Library()
+
+
+@register.filter(name='commonmark')
+def commonmark_filter(value):
+    """
+    Run a string through commonmark.
+
+    Note: This will mark the string as safe. Since commonmark may contain
+    arbitrary HTML, make sure that the input is safe, before applying this
+    filter.
```

Comment:
Well do we make sure that the input is safe?

Reply:
The person that adds a crowdfunding entry will pinky-swear that they don't add malicious HTML to the description 😉